docs/_data/navigation.yml

```diff
@@ -43,6 +43,8 @@
       url: "/housekeeping-git-requests"
     - title: "API Requests"
       url: "/housekeeping-api-requests"
+    - title: "Personal Non-Owner Push"
```

Comment:
Merriam-Webster considers “nonowner” [a proper word](https://www.merriam-webster.com/dictionary/nonowner).

Aside from this, could we rename this to “Personal Repositories with Nonowner Pushes”? I think the title is a bit hard to get otherwise.

Reply:
Any idea for something shorter? Otherwise the menu might look weird